src/guide/migration/introduction.md

```diff
@@ -118,6 +118,7 @@ The following consists a list of breaking changes from 2.x:
 - [`propsData` option](/guide/migration/props-data.html)
 - `$destroy` instance method. Users should no longer manually manage the lifecycle of individual Vue components.
 - Global functions `set` and `delete`, and the instance methods `$set` and `$delete`. They are no longer required with proxy-based change detection.
+- `binding.expression` in [Custom Directives](/guide/migration/custom-directives.html).
```

Comment:
I'm unclear about the link. I think most readers would expect the linked page to provide further information about the removal of `binding.expression` but it isn't mentioned on that page.